internal/controller/flow_falco.go

```diff
@@ -8,13 +8,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/golang/protobuf/ptypes/timestamp"
 	pb "github.com/illumio/cloud-operator/api/illumio/cloud/k8sclustersync/v1"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 // FalcoEvent represents the network information extracted from a Falco event.
 type FalcoEvent struct {
 	// Time is the time the network event occured. ISO 8601 format
-	Time string `json:"time"`
+	Time *timestamppb.Timestamp `json:"time"`
```

Comment:
You still need to send the original `time` field, so preserve it as well for now.
```suggestion
	Time string `json:"time"`
	Timestamp *timestamppb.Timestamp `json:"timestamp"`
```

Reply:
Resolved this